packages/orbit-components/src/primitives/TooltipPrimitive/index.tsx

```diff
@@ -61,17 +35,29 @@ const TooltipPrimitive = ({
 
   const tooltipId = useRandomId();
 
+  const handleEsc = React.useCallback(
+    (ev: { key: string }) => {
+      if (ev.key === "Escape") {
+        setShown(false);
+        setRenderWithTimeout(false);
```

Comment:
Out of curiosity, why isn't `document.removeEventListener("keydown", handleEsc);` added also here? I can call the `handleEsc` function, although the tooltip is no longer visible.

https://github.com/user-attachments/assets/c5d5aa32-d9df-4ad6-a6f4-e18620e118c2

Reply:
Well spotted. Changed